pypowsybl/network/impl/network.py

```diff
@@ -416,6 +429,7 @@ def get_network_area_diagram(self, voltage_level_ids: Union[str, List[str]] = No
             low_nominal_voltage_bound: low bound to filter voltage level according to nominal voltage
             nad_parameters: parameters for network area diagram
             fixed_positions: optional dataframe used to set fixed coordinates for diagram elements. Positions for elements not specified in the dataframe will be computed using the current layout.
+            branch_labels: optional dataframe used to set the labels on a branch
```

Comment:
next step should be the other parameters of the `LabelProvider`: a dataframe with buses description (bus id / description), a dataframe with voltage level description (vl id / description / details). And then comes the dataframe for the styles. Should we encapsulate all these dataframes in a bigger config object, or would it be ok to have these four dataframes as input at the end?